src/org/jgroups/protocols/TP.java

```diff
@@ -443,7 +446,10 @@ protected TP() {
     }
 
     public MsgStats                getMessageStats()     {return msg_stats;}
-    public MessageProcessingPolicy msgProcessingPolicy() {return msg_processing_policy;}
+
+    public MessageProcessingPolicy msgProcessingPolicy()                          {return msg_processing_policy;}
```

Comment:
Very inconsistent naming. The method right above uses proper 'getMessageStats' there's no reason to shorten the name here.

Reply:
Same as comment above (but "get" instead of "is").

Reply:
Existing code, I only formatted it.

Reply:
Updated according to comment.